code/controllers/subsystem/SSweather.dm

```diff
@@ -40,6 +40,9 @@ SUBSYSTEM_DEF(weather)
 		next_hit_by_zlevel["[z]"] = world.time + randTime + initial(W.telegraph_duration)
 
 /datum/controller/subsystem/weather/Initialize()
+	if(!GLOB.configuration.general.enable_default_weather_events)
```

Comment:
Might not be a bad idea to add a debug log?

Reply:
Done, PTAL